data/json/items/book/cooking.json

```diff
@@ -443,6 +443,41 @@
     "fun": -1
   },
   {
+    "id": "vinegar_maker",
+    "type": "BOOK",
+    "name": { "str": "The Vinegar Maker's Handbook", "str_pl": "copies of Vinegar Maker's Handbook" },
+    "description": "This book describes in detail all the ways that one can make vinegar.  You would have that there wasn't much to making vinegar, but the girth of this book tells you otherwise .",
```

Comment:
Whoops
```suggestion
    "description": "This book describes in detail all the ways that one can make vinegar.  You would have that there wasn't much to making vinegar, but the girth of this book tells you otherwise.",
```